dissect/target/target.py

```diff
@@ -310,7 +311,7 @@ def _find(find_path: Path, parsed_path: Optional[urllib.parse.ParseResult]):
                     except Exception as e:
                         getlogger(sub_entry).error("Failed to load target with loader %s", ldr, exc_info=e)
 
-                    if include_children:
+                    if target and include_children:
```

Comment:
```suggestion
                    if target is not None and include_children:
```
Bit clearer perhaps.